scripts/globals/status.lua

```diff
@@ -2709,3 +2699,15 @@ dsp.emoteMode =
     TEXT = 1,
     MOTION = 2
 }
+
+------------------------------------
+-- Skills
```

Comment:
These are A) not skills and B) did not need moved to add a new line

Reply:
This is fixed